lib/examples/CTestTestfile.cmake

```diff
@@ -0,0 +1,9 @@
+# CMake generated Testfile for 
```

Comment:
also this file has been generated by cmake...

Reply:
Removed